### ISSUE TITLE
Proposal: Remove requirement for VCRTForwarders

### ISSUE THREAD

**Issue by sylveon** [microsoft/microsoft-ui-xaml#2459]
# Proposal: Remove requirement for VCRTForwarders

## Summary
Currently, usage of WinUI in non-UWP scenarios requires carrying a set of DLLs known as the VCRT forwarders, which forward methods from the app CRT (which is unavailable in non-UWP, even if packaged) to the corresponding desktop CRT methods. This actually breaks unpackaged .NET scenarios, and is generally not useful. They should be removed.

After conversation with @stevenbrix on Discord, it has come my attention that there's existing internal desire to simplify CRT dependencies and avoid WinUI imposing any CRT requirement, but no formal work item filled.

## Rationale
* It doesn't break unpackaged .NET scenarios (see https://github.com/dotnet/runtime/issues/3735)
* Simplifies adoption of WinUI in XAML islands and islands in general (avoids hard to track DLL not found errors if the forwarders are omitted, so one less pitfall)
* Makes deployment process less complex
* Reduces package size

**Comment by ranjeshj** [comment]
@Austin-Lamb @MikeHillberg as FYI

**Comment by stevenbrix** [comment]
@Scottj1s and @stwish-msft for more info on some plans for the CRT. My understanding is that there are plans to make the VCRTForwarders package no longer needed.

**Comment by Scottj1s** [comment]
Yes, plans are being developed.  Will report more details here when available.

**Comment by sylveon** [comment]
Awesome!

**Comment by taenri** [comment]
Hi @Scottj1s - My team is also interested in this for our project. What is the current status on this issue?
